packages/utils/src/getUniqueSurveyQuestionFileName.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+import generateUUID from 'bson-objectid';
+
+/**
+ * @param {string} originalFileName
+ * @returns {string} unique file name
+ *
+ * This function is used to generate a unique file name for a survey file upload question. It is used by DataTrak and eventually will be used by MediTrak app
+ */
+export const getUniqueSurveyQuestionFileName = originalFileName => {
```

Comment:
I haven't added this into MediTrak App because it looks like the file ID on it's own is used for queueing up files to upload, but this is ready to use once it doesn't do that